boards/arm/nrf5340_dk_nrf5340/nrf5340_dk_nrf5340_shared_sram_planning_conf.dts

```diff
@@ -18,9 +18,9 @@
 	};
 
 	/* SRAM allocated to shared memory */
-	sram0_shared: memory@20010000 {
+	sram0_shared: memory@20070000 {
```

Comment:
Have you verified that ```DT_IPC_SHM_BASE_ADDRESS ``` define points to the new address of shared memory? This DTS definition is used in OpenAMP configuration:

https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/bluetooth/hci/rpmsg_nrf53.c#L32

Reply:
@kapi-no the following is, now, auto-generated:
```
#define DT_IPC_SHM_BASE_ADDRESS                     0x20070000
#define DT_IPC_SHM_SIZE                             64
```

As you are the author of the rpmsg-based LE stack split, can you confirm that the solution still works when relocating the area of shared SRAM?

Reply:
Yes, it should still work.